Wrappers/Python/cil/optimisation/functions/TotalVariation.py

```diff
@@ -148,18 +161,19 @@ def __init__(self,
                  isotropic = True,
                  split = False,
                  info = False, 
-                 strong_convexity_constant = 0):
+                 strong_convexity_constant = 0,
+                 warmstart=False):
```

Comment:
```suggestion
                 warmstart=True):
```

Reply:
I suppose that we will need to change a long list of unit tests to account for this new default.

Reply:
I am on it - @gfardell has set me up!

Reply:
Hi @gfardell - had a bit of a play with unit tests. Happy to discuss it tomorrow!